cdc/scheduler/internal/v3/keyspan/reconciler.go

```diff
@@ -30,6 +30,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const spanRegionLimit = 50000
```

Comment:
why you choose 5000 ? please add some comment.

Reply:
We don't want a span to be too large or too small. Currently, this value is inferred based on user scenarios.